src/front/js/component/Pricing.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "../../styles/Pricing.css";
+
+const Home = () => {
+  return (
+    <React.Fragment>
+      <section className="content-container">
+        <div className="columns">
```

Comment:
Hmm... Those divs with "columns" classnames... They're so similar! Why don't we transform them into components and pass their differences through `props`? 🤓

Reply:
I like this idea, but remember we have 3 weeks left now and we need to have the Minimum Value Product. We can create a card for this in Trello as an improvement, wdty? :)